### PR TITLE
Update HDRP frame diagram + TAA bit in the custom post process doc

### PR DESCRIPTION
**Pull request by alelievr**
### Purpose of this PR
Update the HDRP frame diagram image + add a bit of text about TAA in the custom post process page.

### CODE REVIEW

**Review by sebastienlagarde — APPROVED** [review]

File: com.unity.render-pipelines.high-definition/Documentation~/Custom-Post-Process.md [hunk 1]
```diff
@@ -92,6 +92,8 @@ The **injectionPoint** override allows you to specify where in the pipeline HDRP
 For more detailed information on where HDRP injects the custom post-process passes, see the following diagram:
 ![](Images/HDRP-frame-graph-diagram.png)
 
+Note that if TAA is enabled, it will be applied between the injection points **BeforeTAA** and **beforePostProcess**. When using the physically based [Depth Of Field](Dept-Of-Field.md) is used, a second pass of TAA will be applied for the temporal accumulation of the physically based Depth Of Field.
+
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Documentation~/Custom-Post-Process.md:96]: Currently in the diagram, now we have merge decal emissive into the forward opaque passe. So we should rename the pass: Forward Opaque + Decal Emissive + Emissive for deferred Material. Thanks

Comment (Vic-Cooper) [reply com.unity.render-pipelines.high-definition/Documentation~/Custom-Post-Process.md:96]: Phrasing suggestion: 

**Note**: When you enable [Temporal anti-aliasing (TAA)](Anti-Aliasing.md#TAA), HDRP applies TAA between the injection points **BeforeTAA** and **beforePostProcess**. When you use [Depth Of Field](Post-Processing-Depth-of-Field.md) and enable its **Physically Based** property, HDRP performs a second TAA pass to perform temporal accumulation for this effect.

Comment (alelievr) [reply com.unity.render-pipelines.high-definition/Documentation~/Custom-Post-Process.md:96]: I update the doc and the diagram: https://app.diagrams.net/#G1zKeFASXBg4mo6_GfzkPUav7U2tIx60LU

**Review by Vic-Cooper — APPROVED** [review]
Great, done :)

**Pull request merged**